recipes-core/netifd/netifd_git.bb

```diff
@@ -56,10 +56,6 @@ do_install_append() {
     install -dm 0755 ${D}/etc/modules.d ${D}/etc/modules-load.d
     echo "bridge" >${D}/etc/modules.d/30-bridge
     echo "bridge" >${D}/etc/modules-load.d/bridge.conf
-
-    # Remove duplicate files under /lib/
-    rm -rf ${D}/lib/
-
```

Comment:
Please make it conditional on bitness

Reply:
@kraj Recipe is already adapted to use "${base_libdir}" to select between lib/lib64 in all places. So "rm -rf ${D}/lib/" removes the actual content which is copied for 32 bit machine only. 

This means we need "lib" or "lib64", which is selected with ${base_libdir}. So this removal is not required for both 32 and 64 platforms.

Reply:
@kraj Tested Ok in qemu 64 and rpi3